src/components/BogTooltip/styles.module.css

```
@@ -0,0 +1,20 @@
+@layer base {
+    .bog-tooltip__content {
+        background-color: var(--color-grey-fill-weak);
+        color: var(--color-grey-text-strong);
+        border: none;
+        border-radius: 0.6rem;
+        padding: 0.5rem 0.5rem;
+        font-family: var(--font-paragraph);
+        font-size: var(--text-mobile-paragraph-2);
+        line-height: var(--text-mobile-paragraph-2--line-height);
```

```suggestion
        font-size: var(--text-mobile-small);
        line-height: var(--text-mobile-small--line-height);
```

I think the smaller text makes it look more like the Figma.